packages/frontend/src/components/HoursWithdrawal.vue

```diff
@@ -251,6 +255,7 @@ export default Vue.extend({
     Input,
     CenterColumnWrapper,
     OvertimeVisualizer,
+    NoHolidayData: () => import("./NoHolidayData.vue"),

```

Comment:
Tror ikke det er noe behov for å lazy-loade denne komponenten. Kan bli erstattet med `NoHolidayData`